diff --git a/node_modules/nano/lib/nano.d.ts b/node_modules/nano/lib/nano.d.ts
index c816606..2e73902 100644
--- a/node_modules/nano/lib/nano.d.ts
+++ b/node_modules/nano/lib/nano.d.ts
@@ -1665,7 +1665,7 @@ declare namespace nano {
      * By default the information returned contains only the document ID and revision. */
     rows: Array<{
       id: string;
-      key: string;
+      key: any;
       value: V;
       doc?: D & Document;
     }>;
